### PR TITLE
FEATURE: allow to hide links for anon or non-staff users

### PR DESCRIPTION
**Pull request by AlexP11223**
would be better to allow this for each link separately, but I am not sure what's the best way to implement this with the current input format

### CODE REVIEW

File: common/common.scss [hunk 1]
```diff
@@ -37,3 +37,17 @@
     }
   }
 }
+
+@if $min_user_level == user {
```
Comment (ZogStriP) [inline common/common.scss:41]: Don't think it's a good idea to add conditions in the scss files (if it even works?)

Comment (AlexP11223) [reply common/common.scss:41]: dunno, but there is already condition for `left`.

It's quite simple, it just hides the whole block if the staff/user class is present.

Comment (ZogStriP) [reply common/common.scss:41]: We already have `anon` and `staff` class on the `body` / `html` elements. Why not use that instead?

Comment (AlexP11223) [reply common/common.scss:41]: I think that's exactly what I did?

Comment (ZogStriP) [reply common/common.scss:41]: Right... Then why the need for the `@if` ?

Comment (AlexP11223) [reply common/common.scss:41]: It's a component option that specifies when the links should be displayed. Very similar to the position option.

**Comment by ZogStriP** [comment]
Yes, would be much better to add the option on a per-link basis.

**Pull request closed without merging**